bin/lisky

```diff
@@ -16,6 +16,9 @@
  */
 var os = require('os');
 var lockfile = require('lockfile');
+var semver = require('semver');
+
+var packagejson = require('../package.json');
```

Comment:
Is JSON a relevant part of the name? Usually I would say the name shouldn't include the extension, but maybe in this case it's such an intrinsic part of how people refer to this file in a project?

Reply:
In this case, `package` is reserved word in JS, so I would say `packagejson` => `packageJSON`